src/System.Net.Http/ref/System.Net.Http.csproj

```diff
@@ -3,6 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{132BF813-FC40-4D39-8B6F-E55D7633F0ED}</ProjectGuid>
+    <!-- in most cases the desktop configuration of a contract is just for APICompat,
+         however in this case it is the actual ref we want to ship -->
+    <IncludeDesktopRefInPackage>true</IncludeDesktopRefInPackage>
```

Comment:
You're already building a NET configuration for `src/System.Net.Http/ref/System.Net.Http.csproj` but before this was being excluded from the package and folks would get the previous built version.  This is a bug compared to 1.x codebase which caused by the way packaging/traversal was re-implemented in 2.x corefx repo.  This change simply fixes the bug.

This merely includes the dll which we're already building.  If you'd like for that to be different in any way simply ifdef the source.